rstudio/c9s-python-3.9/Dockerfile

```diff
@@ -50,9 +50,17 @@ COPY rsession.conf /etc/rstudio/rsession.conf
 # package installation 
 RUN dnf install -y libsodium-devel.x86_64 libgit2-devel.x86_64 libcurl-devel harfbuzz-devel.x86_64 fribidi-devel.x86_64 cmake "flexiblas-*" \
     && dnf clean all && rm -rf /var/cache/yum
-RUN R -e "install.packages('Rcpp')"
-
-# Install NGINX to proxy RStudio and pass probes check 
+# Installation of R Packages 
+RUN R -e "install.packages('remotes')"
+RUN R -e "require('remotes') && \
+remotes::install_versions('Rcpp','1.0.12' ) && \
+remotes::install_versions('tidymodels', '2.0.0') && \
+remotes::install_versions('tidyverse','1.1.1') && \
+remotes::install_versions('plumber','1.2.1') && \
+remotes::install_versions('vetiver','0.2.5') && \
+remotes::install_versions('devtools','2.4.5')"
```

Comment:
```suggestion
# Install R packages
RUN R -e "install.packages('remotes')"
RUN R -e "require('remotes'); \
    remotes::install_version('Rcpp','1.0.12'); \
    remotes::install_version('tidyverse','2.0.0'); \
    remotes::install_version('tidymodels','1.1.1'); \
    remotes::install_version('plumber','1.2.1'); \
    remotes::install_version('vetiver','0.2.5'); \
    remotes::install_version('devtools','2.4.5');"
```

In R , seems like `;` would do the trick instead of  `&&`